demo/firebase-messaging.html

```diff
@@ -60,6 +60,9 @@
       window.addEventListener('WebComponentsReady', function() {
         if (navigator.serviceWorker) {
           navigator.serviceWorker.register('firebase-messaging-sw.js').then(function(sw) {
+            var app = document.getElementById('app');
+            // polymer 2.0 compatibility
+            app = app.$ ? app : app.parentNode;
```

Comment:
Huh, how? Isn't `$` in both things? Maybe you need `Polymer.Element` somewhere instead